Aga.Controls/Tree/NodeControls/BaseTextControl.cs

```diff
@@ -219,23 +219,23 @@ private static Brush GetFrush(Color color)
 
 		private void CreateBrushes(TreeNodeAdv node, DrawContext context, string text, out Brush backgroundBrush, out Color textColor, out Font font, ref string label)
 		{
-			textColor = SystemColors.ControlText;
+            textColor = Parent.ForeColor;
```

Comment:
The formatting in this entire file is off.

Reply:
Yes, formatting is way off on this file (I assumed that was on purpose?). I can fix the formatting, but that's basically going to change the whole file, should I do that?

As for the other files (under LibreHardwareMonitor), I'm using VS2022 and have tried VSCode as well, both seem to be loading the .editorconfig, but I'm not getting a different formatting, what am I missing?

Reply:
I've created [#847](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/847) that has this code, with fixed merge conflicts, and another nice to have for dark theme (thicker plot lines)